Localize/LocProject.json

```diff
@@ -0,0 +1,31 @@
+{
+	"Projects": [
+	  {
+		"LanguageSet": "Azure_Languages",
+		"LocItems": [
+		  {
+			"SourceFile": "./msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx",
```

Comment:
We should not use relative paths but full paths. Will make the CI more robust. You can use 'sed' to replace variables in an LocProject.json.in file for example like we do here: https://github.com/xamarin/xamarin-macios/blob/main/tools/devops/Makefile#L4 and we add it as part of the make all